Examples/Python/python/acts/examples/reconstruction.py

```
@@ -285,6 +285,7 @@ def addSeeding(
     spacePointGridConfigArg: SpacePointGridConfigArg = SpacePointGridConfigArg(),
     seedingAlgorithmConfigArg: SeedingAlgorithmConfigArg = SeedingAlgorithmConfigArg(),
     houghTransformConfig: acts.examples.HoughTransformSeeder.Config = acts.examples.HoughTransformSeeder.Config(),
+    adaptiveHoughTransformConfig: acts.examples.AdaptiveHoughTransformSeeder.Config = acts.examples.AdaptiveHoughTransformSeeder.Config(),
```

Ruff (0.8.2)

Do not perform function call `acts.examples.AdaptiveHoughTransformSeeder.Config` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)